0000-template.md

```diff
@@ -1,51 +1,48 @@
+- Title: (fill me in with a human-focussed title for the RFC)
 - Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
-- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- RFC PR: [#0000](https://github.com/rust-lang/rfcs/pull/0000)
```

Comment:
```suggestion
- RFC PR: https://github.com/rust-lang/rfcs/pull/0000
```

Reply:
These won't render as links in [the rendered book](https://rust-lang.github.io/rfcs/) without angle brackets.

Reply:
Ah, fair enough. Editing.

Reply:
Why do you prefer this syntax? ISTM that the original and the proposed change are GitHub-standard and clear and concise. Spelling out the whole link is less readable.

Reply:
I think in practice people are just going to click on the link. Also the number is perfectly readable at the end of the URL, but it's nice to be able to see the repository at a glance. I'm trying to preserve the information that's already available in the existing version.

The existing version, with `rust-lang/rfcs#0000`, provides enough information to know at a glance where it's going; the URL also preserves that information. Just `#0000` doesn't.

Reply:
The usual GitHub convention is to use #nnn for issues and PRs in the same repo and org/repo#nnn for issues and PRs in other repos. That is what I was following here. (Note also, that since this is a link, the browser will show the URL on hover, and the fact that this is an RFC PR is shown on the left of the colon). If there is consensus that we need the repo info then I'd be happy to drop this change (it is very minor, after all). IMO having the full link is less readable than either the proposed or former versions.